test/functional/README.md

```diff
@@ -71,23 +71,23 @@ P2P messages. These can be found in the following source files:
 
 #### Using the P2P interface
 
+- Set `setup_mininode_event_loop = True` in `set_test_params()` to spawn a thread
```

Comment:
Would it be possible to start the network thread for all tests, so it's not necessary for the test writer to even perform this step?

Reply:
Good point. I guess there is no overhead in running an event loop with no events.